### PR TITLE
Styles update in background.scss popup.scss and little refactor

### PR DESCRIPTION
**Pull request by Pachciar3**
- I fixed a problem with white colours in some elements
- Elements now use different font colours, this is important update for User Experience
- I changed the colours for the switch in the popup
- I also did a small refactor

@MennoVK , please give me a review ;-)

### CODE REVIEW

**Comment by MennoVK** [comment]
Also in the manifest file the description is: "It is a dark theme for the udemy website". It would be better English to just say: "A dark theme for the udemy website".

**Comment by MennoVK** [comment]
I don't know if this is the way i am supposed to do the review, but i am sure you can give me feedback :).

File: styles/background.scss [hunk 1]
```diff
@@ -181,12 +224,24 @@ html body.dark_mode_on {
...
   .ud-btn-secondary:active,
-  .ud-btn-secondary:hover {
+  .ud-btn-secondary:hover,
+  .item-card-module--item-card-title--S729p:hover {
     color: var(--hover-font-color);
   }
-  .ud-footer .inverted-link {
+
+  .ud-footer .inverted-link,
+  .smart-bar-module--smart-bar__title--TnwIQ,
```
Comment (MennoVK) [inline styles/background.scss]: The smartbar on top of the page changes colors a lot. Right now for me the bar is purple with a purple text color. Therefore, it is hard to read. As far as i have seen, it is sometimes purple and sometimes yellow, but i don't know if that is all colors.

File: styles/options.scss
Comment (MennoVK) [inline styles/options.scss:1]: This is not a change you made for this version, but at line 97 at the "&--success" tag, the color of text on hover is yellow. This seems a bit out of theme and also out of line wiht the popup where there is only an underline added on hover. Maybe keep these 2 styles even or change the color to something more in line with the other styles.

File: styles/background.scss
Comment (MennoVK) [inline styles/background.scss:1]: Something i ran into myself is that after a video ends there is a small bar that pops up with the text: "how did this section go". but the background of that is in white. Is it possible to add code when i notice something like this while reviewing? Do i add it, do you add it or do we add it to a later version? The selector for it is: ".fast-feedback--container--1mut2".

**Review by MennoVK — CHANGES_REQUESTED** [review]
I don't know if this is the way i am supposed to do the review, but i am sure you can give me feedback :). I added a few comments on things i think are relevant. Otherwise, great work!

**Comment by Pachciar3** [comment]
> 
Yes this good way, thanks.  The goal of the review is to identify issues that reduce code quality, catch bugs and make sure every collaborator knows what is being changed.

**Comment by Pachciar3** [comment]
> Also in the manifest file the description is: "It is a dark theme for the udemy website". It would be better English to just say: "A dark theme for the udemy website".

Thanks because I do not speak English very well

File: styles/background.scss (hunk 1 above)
Comment (Pachciar3) [reply styles/background.scss]: Can you give me a screenshot of smartbar ?

File: styles/options.scss
Comment (Pachciar3) [reply styles/options.scss:1]: The color will be yellow when user change theme color with success result. This is good behaviour and good for user experience

Comment (Pachciar3) [reply styles/options.scss:1]: I think you think about links in the footer. I changed ;-)

Comment (MennoVK) [reply styles/options.scss:1]: No i meant this:
image
These links turn yellow on hover. Is that expected behaviour?

Comment (MennoVK) [reply styles/options.scss:1]: Yes correct!

**Comment by MennoVK** [comment]
> 

No worries. I can understand completely fine, but like this it is a little better : ).

File: styles/background.scss (hunk 1 above)
Comment (MennoVK) [reply styles/background.scss]: image
As you can see it is currently yellow. It changes from time to time. Right now the textcolor is purple and with a purple background it is hard to read. If i run into it when it is purple i will send another image.

File: styles/background.scss
Comment (Pachciar3) [reply styles/background.scss:1]: 1. I can't found that problem, but I added this class. Please test 
2. I found other problem and I added some classes too
Screenshot

After
Screenshot

Comment (MennoVK) [reply styles/background.scss:1]: 1. Yeah that fixed it.
2. Ah, great. I ran into that as well.

File: styles/background.scss (hunk 1 above)
Comment (Pachciar3) [reply styles/background.scss]: image
I found the code with all variants. I repaired and tested, I think now it should be easier to read in all variants

**Comment by Pachciar3** [comment]
You have to accept the pull request when you think all issues all resolved :-)

**Review by MennoVK — APPROVED** [review]

**Pull request merged**